src/utils.js

```diff
@@ -8,7 +8,7 @@ export function restoreToken({ walletAddress }) {
       ? JSON.parse(window.localStorage.getItem('tokens'))
       : {};
     return tokens[walletAddress];
-  } catch (e) {
+  } catch {
```

Comment:
maybe log error? Or it is not relevant?

Reply:
Added  ` console.error('Restore token error:', e);`